pkg/ddc/juicefs/replicas.go

```diff
@@ -17,131 +17,39 @@ limitations under the License.
 package juicefs
 
 import (
-	"context"
-	"reflect"
-
-	corev1 "k8s.io/api/core/v1"
+	"github.com/fluid-cloudnative/fluid/pkg/utils/kubeclient"
 	"k8s.io/client-go/util/retry"
 
-	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
-	"github.com/fluid-cloudnative/fluid/pkg/common"
 	cruntime "github.com/fluid-cloudnative/fluid/pkg/runtime"
-	"github.com/fluid-cloudnative/fluid/pkg/utils"
 )
 
 // SyncReplicas syncs the replicas
-func (j *JuiceFSEngine) SyncReplicas(ctx cruntime.ReconcileRequestContext) error {
-	runtime, err := j.getRuntime()
+func (j *JuiceFSEngine) SyncReplicas(ctx cruntime.ReconcileRequestContext) (err error) {
+	var (
+		workerName string = j.getWorkerName()
+		namespace  string = j.namespace
+	)
+
+	workers, err := kubeclient.GetStatefulSet(j.Client, workerName, namespace)
```

Comment:
I suggest put Line 33 into RetryOnConflict method.
As RetryOnConflict doc says 
//     err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
//         // Fetch the resource here; you need to refetch it on every try, since
//         // if you got a conflict on the last update attempt then you need to get
//         // the current version before making your own changes.
//         pod, err := c.Pods("namespace").Get(name, xxx)
//         if err ! nil {
//             return err
//         }